sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -0,0 +1,293 @@
+## API Report File for "@azure/monitor-query"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+import { OperationOptions } from '@azure/core-http';
+import { PipelineOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export type AggregationType = "None" | "Average" | "Count" | "Minimum" | "Maximum" | "Total";
+
+// @public
+export interface BatchQuery {
+    azureResourceIds?: string[];
```

Comment:
whenever we have an array property, it's generally preferred that we initialize to the empty array if possible - is that possible for any properties on this type?

Reply:
This is a request type (not a response). Are you saying the caller should pass in empty arrays rather than just omitting the field (which is the intent here)

Reply:
nah, I thought it was an output. Input is better this way.